controllers/manager/monitor.go

```diff
@@ -425,8 +425,40 @@ func ManageStrategy(management CloudManager) bool {
 			log.Info("Vim client is not ready. Wait")
 			return false
 		} else {
+
+			strategy_status, err := management.GcShow(client)
```

Comment:
Can we move this section to a new method and give proper comment? Hard to read this block tbh?

Reply:
Done